src/AutomatedCar/Models/World.cs

```diff
@@ -20,13 +30,184 @@ public AutomatedCar ControlledCar
             set => this.RaiseAndSetIfChanged(ref this._controlledCar, value);
         }
 
+        private List<WorldObject> trees;
+        private List<Sign> signs;
+        private List<WorldObject> roads;
+        private List<WorldObject> npcs; // for 2nd sprint
```

Comment:
Ezzel nem értek egyet, miért jó ez?
Miért kellett a közös WorldObject gyűjteményt többre cserélni, ami mind WorldObject, kivéve a Sign (ez még akkor is hiba, ha az elv oké).

Megnéztem a user story-t és látom, hogy az idei verzióból kimaradt egy sor, ami kb. az volt, hogy „heterogén gyűjtemény biztosítása a világelemekhez”, ami azt jelentette, hogy egyetlen közös típusgyűjteménnyel el lehessen érni az összes világobjektumot.

Ettől eltekintve a Skeleton ezt így csinálta. Ettől eltértek. Mem az a bajom, hogy eltértek tőle, hanem az, hogy nem látom, hogy ez miért jó. Ha erre sikerül kielégítő magyarázatot adni, akkor el fogjuk fogadni, de a modell tervben sem szerepelt ez így.